src/plugins/data/public/query/query_string/query_string_manager.ts

```diff
@@ -166,6 +166,7 @@ export class QueryStringManager {
     const language = this.languageService.getLanguage(languageId);
     const dataset = curQuery.dataset;
     const input = language?.getQueryString(curQuery) || '';
+    this.languageService.setUserQueryString(input);
```

Comment:
Why do we need this change?

Reply:
Need to get query from local storage for reloading the page

Reply:
Sounds good, thanks :)